server/datastore/mysql/hosts.go

```diff
@@ -5196,6 +5196,18 @@ func numHostsFleetDesktopEnabledDB(ctx context.Context, db sqlx.QueryerContext)
 	return count, nil
 }
 
+func numHostsABMPendingDB(ctx context.Context, db sqlx.QueryerContext) (int, error) {
+	var count int
+	const stmt = `
+		SELECT count(*) FROM hosts WHERE hosts.osquery_host_id IS NULL
```

Comment:
We need to filter out iPhones/iPads/Androids:
```suggestion
		SELECT COUNT(*) FROM fleet.hosts WHERE osquery_host_id IS NULL AND platform NOT IN ('ios', 'ipados', 'android');
```

Reply:
Thanks! Committed.

Reply:
Should we not remove WIndows and Linux as well?

Reply:
No need. Those have osquery.